sdk/communication/azure-communication-rooms/pom.xml

```diff
@@ -64,7 +64,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-communication-common</artifactId>
-      <version>1.3.0-beta.1</version> <!-- {x-version-update;com.azure:azure-communication-common;current} -->
+      <version>1.2.1</version> <!-- {x-version-update;com.azure:azure-communication-common;current} -->
```

Comment:
We're keeping this to avoid breaking the ACS Rooms build. Will change to use a dependency tag after ACS Common releases.

/cc @ramneettung-msft @elavarasidc